app/javascript/actionsPanel/initializeActionsPanelToggle.js

```diff
@@ -28,9 +28,11 @@ export function initializeActionsPanel(user, path) {
     const modContainer = document.getElementById('mod-container');
     const panelDocument = modContainer.contentDocument;
 
-    panelDocument
-      .getElementsByClassName('close-actions-panel')[0]
-      .classList.remove('hidden');
+    if (panelDocument) {
```

Comment:
This was causing a mostly harmless error when not present, but Cypress took exception to it 😅